.github/workflows/issue-comment.yml

```diff
@@ -23,6 +23,12 @@ jobs:
     env:
       GH_TOKEN: ${{ secrets.token }}
     steps:
+      - name: Checkout newrelic-php-agent code
+        uses: actions/checkout@v4
+        with:
+          path: php-agent
```

Comment:
Will this be enough for `gh` to pick up which repo it needs to operate with? Wouldn't it be better instead of checkout step to provide repo as an argument to `gh` command (`-R` - see https://cli.github.com/manual/gh_issue_comment)?